### ISSUE TITLE
MySQL monitor

### ISSUE THREAD

**Issue by nitzan-tz** [F5Networks/f5-appsvcs-extension#301]
### Is your feature request related to a problem? Please describe.
We would like to have AS3 deceleration that cover all of our application stack and the missing part is MySQL monitor 

### Describe the solution you'd like
I would like to have MySQL monitor like you have for DNS,FTP and HTTP

### Describe alternatives you've considered
We would like to have the full configuration via declaration so we do not have a lot of choices

### Additional context
This is the current MySQL monitor we will need all these variables 
```
ltm monitor mysql MYSQL_MON {
    count 1
    database test
    debug no
    defaults-from mysql
    destination *:*
    interval 9
    password ***************
    recv good
    recv-column 1
    recv-row 1
    send "select test.get_galera_status();"
    time-until-up 0
    timeout 10
    username mysql_user
}
```

**Comment by dstokesf5** [comment]
I have added this issue to our internal product backlog as AUTOTOOL-1714.

You mention needing to configure the defaults-from property. As a general design principal, AS3 does not expose that property on profiles to avoid source of truth confusion. Will your use case still be satisfied without the defaults-from property being exposed through an AS3 declaration?

**Comment by nitzan-tz** [comment]
Its ok not to have defaults-from but please expose all options so we will able to configure everything in the deceleration

**Comment by delgadillo22** [comment]
Added to the backlog as https://jira.pdsea.f5net.com/browse/AUTOTOOL-1714 for prioritization

**Comment by dstokesf5** [comment]
This was resolved in the 3.23.0 release.
